features/step_definitions/user_accounts.rb

```diff
@@ -96,7 +97,7 @@
   employee_user = User.where(:'roles'.in => ["employee"], locked_at: nil).first.oim_id
   employer_user = User.where(:'roles'.in => ["employer_staff"], locked_at: nil).first.oim_id
   broker_user = User.where(:'roles'.in => ["broker"], locked_at: nil).first.oim_id
-  expect(page).not_to have_content(employee_user)
+  expect(page).to have_content(employee_user)
```

Comment:
we are seeing that employer since unlocked user is going to have attribute locked_at => nil 
when we query we will get that record.